ui/components/ui/text-field/index.scss

```diff
@@ -0,0 +1,4 @@
+.MuiInput-input,
+.MuiInputBase-input {
+  color: var(--color-text-default) !important;
```

Comment:
I resorted to adding a stylesheet to set the color of these input elements.  We absolutely must get rid of this https://github.com/MetaMask/metamask-extension/blob/develop/ui%2Fcomponents%2Fui%2Ftext-field%2Ftext-field.component.js#L13-L85 nonsense in the future.